std/algorithm/mutation.d

```diff
@@ -1940,6 +1944,100 @@ if (isBidirectionalRange!Range
             [ 1, 3, 3, 4, 5, 5, 6 ]);
 }
 
+@nogc unittest
+{
+    // @nogc test
+    int[10] arr = [0,1,2,3,4,5,6,7,8,9];
+    static bool pred(int i)
+    {
+        if (i < 5)
+            return true;
+        return false;
+    }
+    auto r = arr[].remove!(SwapStrategy.unstable)(0);
+    r = r.remove!(SwapStrategy.stable)(0);
+    r = r.remove!(pred, SwapStrategy.unstable);
+    r = r.remove!(pred, SwapStrategy.stable);
+}
+
+unittest
```

Comment:
FYI: (not important for now) in the near future there might be a new check that ensures that all unittests have either a `@safe` or a `@system attribute, because it helps to avoid [regressions](https://github.com/dlang/phobos/pull/4947/files#r92094983).